scripts/benchmark_inference.py

```diff
@@ -146,7 +147,7 @@
 print("loading model")
 model = models.get_model(args.architecture, args.variant, device_type=args.device_type)
 tokenizer = tokenizers.get_tokenizer(args.tokenizer)
-
+print(torch.cuda.memory_allocated(device) /1024 /1024/1024)
```

Comment:
if we're going to isort should we black too? I think black would put spaces after the `/`'s. 

it might be helpful to include units in the print statement too.

we could also add `scripts` to `sort-paths` in .github/workflows/lint.yml